kirin/core/merge_utils.py

```diff
@@ -360,37 +360,74 @@ def log_stu_modif(trip_update, stu, string_additional_info):
     )
 
 
+def manage_arrival_consistency(stu, trip_update, previous_stop_event):
+    if stu.arrival is not None:
+        return True
+
+    logger = logging.getLogger(__name__)
+
+    stu.arrival = stu.departure
+    if stu.arrival is None and previous_stop_event.time is not None:
+        stu.arrival = previous_stop_event.time
+    if stu.arrival is None:
+        logger.warning(
+            "TripUpdate on navitia vj {nav_id} on {date} rejected: "
+            "StopTimeUpdate missing arrival time".format(
+                nav_id=trip_update.vj.navitia_trip_id, date=trip_update.vj.get_circulation_date()
+            )
+        )
+        return False
+    log_stu_modif(trip_update, stu, "arrival = {v}".format(v=stu.arrival))
+    if not stu.arrival_delay and stu.departure_delay:
+        stu.arrival_delay = stu.departure_delay
+        log_stu_modif(trip_update, stu, "arrival_delay = {v}".format(v=stu.arrival_delay))
+
+    return True
+
+
+def manage_departure_consistency(stu, trip_update):
```

Comment:
For consistency, we should return booleans in departure and arrival checks:
```suggestion
def manage_departure_consistency(stu, trip_update):
"""
check and adjust departure time and delay
returns False is feed is not consistent for that stop, True otherwise (currently always True)
"""
```